### PR TITLE
use only closure actions internally

### LINKED ISSUE #213: Please remove sendAction deprecation warning.
deprecate.js:102 DEPRECATION: You called sendAction("onChange") but Component#sendAction is deprecated. Please use closure actions instead. [deprecation id: ember-component.send-action] See https://emberjs.com/deprecations/v3.x#toc_ember-component-send-action for more details.

### PR DESCRIPTION
**Pull request by H1D**
Another simple PR to get familiarized with codebase.

This PR changes all `this.sendAction(XXX)` to direct invocation `this.XXX()`. Fixes #213 

Notes:

* ember > 1.13 needed (not an issue since we require Ember >=2.4 for `ember-sortable 2.x` )
* `sendAction()` was [deprecated](https://deprecations.emberjs.com/v3.x/#toc_ember-component-send-action) in 3.4 and will be removed in 4.0

### CODE REVIEW

**Comment by PeterLD** [comment]
Is there anything blocking this from being merged / released?

**Comment by jgwhite** [comment]
@PeterLD trying to carve out time to review all the new good stuff landing in 2.x.

File: addon/components/sortable-group.js [hunk 1]
```diff
@@ -13,6 +13,18 @@ export default Component.extend({
...
+    Called when order of items has been changed
+
+    @property onChange
+    @type Action
+    @param object group model (omitted if not set)
+    @param array item models in their new order
+    @param object item model just dragged
+    @default null
+  */
+  onChange: ()=>{},
```
Comment (btecu) [inline addon/components/sortable-group.js]: `onChange: () => { },`

Comment (H1D) [reply addon/components/sortable-group.js]: OK, maybe we can have eslint rule for that?

Comment (jgwhite) [reply addon/components/sortable-group.js]: And incorporate Prettier into the project :)

**Comment by stephencattaneo** [comment]
@H1D Able to make those changes?

@jgwhite / @btecu any other comments?

**Comment by H1D** [comment]
@stephencattaneo OK done

**Comment by stephencattaneo** [comment]
@H1D Looks like you only fixed one of the instances of `()=>{}`. If you rebase off of develop you should now see the linter fails for those empty arrow functions.

**Comment by H1D** [comment]
Oh! OK. Thx

**Comment by gorner** [comment]
Any timeline for merging this in (or for 2.x generally)? My team is trying to clear Ember 3.x deprecations (and we use deprecation-workflow to throw errors to ensure no regressions once we’ve cleared each one). We can defer the sendAction changes or fork for now, but a timeline for release would be helpful.

**Comment by dbashford** [comment]
Add this onto the `volatile` deprecation notices that would be great to have cleaned up.

**Comment by mattgraham1995** [comment]
Will this be reviewed and merged soon? My team is also trying to cleanup the related deprecation warnings in our project.

**Comment by btecu** [comment]
Ping @jgwhite.

**Comment by jgwhite** [comment]
Howdy y’all

We’re on reorganizing the maintenance around this addon so that we can release a v2 effectively as soon as possible. In the meantime, thanks for your patience while we get things together.

**Comment by H1D** [comment]
hmm... that rebase went wrong. Gonna fix it later today

**Comment by H1D** [comment]
@jgwhite @btecu looks OK now

**Comment by ygongdev** [comment]
@H1D 👋 I'm going to merge a bunch of stuff that will break your current change. Do you mind fix the those conflicts later? I would love to merge this in.

**Comment by stephencattaneo** [comment]
If H1D doesn't come through with this, I'll open up a new PR and handle it.

**Comment by H1D** [comment]
@stephencattaneo would be great! I'm a bit busy next few days.

added you as contributor to my fork repo – https://github.com/H1D/ember-sortable so you don't have to re-create PR. Just push new commits to `closure-actions` branch

**Comment by ygongdev** [comment]
@stephencattaneo Let me know if you're already working on this.
If not, I can just quickly fix this up and merge.

**Comment by stephencattaneo** [comment]
@ygongdev   I was hoping to get to this last week. Next time I have for this type of work is Friday. If you're able to move quick on it be my guest :)

**Comment by ygongdev** [comment]
Closing for #282. I saw that there were some contextual component change here as well. I'll include that in a separate commit.

**Pull request closed without merging**